sdk/identity/identity/src/credentials/managedIdentityCredential/imdsMsi.ts

```diff
@@ -76,9 +76,9 @@ function prepareRequestOptions(
   };
 }
 
-// 800ms -> 1600ms -> 3200ms
+// 800ms -> 1600ms -> 3200ms -> 6400ms -> 12800ms
 export const imdsMsiRetryConfig = {
-  maxRetries: 3,
+  maxRetries: 5,
```

Comment:
Any concerns about this being a breaking change? If I'm a customer, I may have bumped my own request timeout to some number based on observable behavior with the current retry count which has now increased...

Reply:
Pushed these changes to a separate PR https://github.com/Azure/azure-sdk-for-js/pull/29139